data/rbac_approval_seed.yml

```diff
@@ -0,0 +1,85 @@
+---
+resource_definition1: &AdminResourceDefinition
+  attributeFilter: 
+    value: '*'
+    key: id
+    operation: equal
+access1: &AdminReadAccessTemplates
+  permission: approval:templates:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access2: &AdminCreateAccessWorkflows
+  permission: approval:workflows:create
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access3: &AdminReadAccessWorkflows
+  permission: approval:workflows:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access4: &AdminUpdateAccessWorkflows
+  permission: approval:workflows:update
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access5: &AdminReadAccessRequests
+  permission: approval:requests:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access6: &AdminReadAccessStages
+  permission: approval:stages:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access7: &AdminReadAccessActions
+  permission: approval:actions:read
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access8: &AdminCreateAccessActions
+  permission: approval:actions:create
+  resourceDefinitions:
+          - *AdminResourceDefinition
+access61: &ApproverCreateAccessActions
+  permission: approval:actions:create
+access62: &ApproverReadAccessActions
+  permission: approval:actions:read
+access63: &ApproverReadAccessStages
+  permission: approval:stages:read
+access64: &ApproverReadAccessRequests
+  permission: approval:requests:read
+role1: &ApprovalAdministratorRole
+  name: Approval Administrator
+  system: true
+  version: 1
+  description: An Approval administrator roles grants create, read and update permissions
```

Comment:
An approval administrator role that grants create, read, and update permissions.